FanTheorem.agda

```diff
@@ -55,8 +53,8 @@ module _ (𝔅 : ℘ (Neigh 𝟚)) (𝔅? : ∀ U → Decidable (𝔅 U)) (⊨
 
   𝔅⊑𝔄 : 𝔅 ⊑ 𝔄
   𝔅⊑𝔄 𝔅[U] =
-    1 , λ α →
-      ze ,
+    1 ▸ λ α →
```

Comment:
Yeah, I should have changed it awhile ago but I couldn't come up with a good replacement and forgot about it. Guess I was overthinking it.